README.md

```diff
@@ -83,6 +83,17 @@ mashery-toolbelt promote 8t7a4qwh2dgk97tjmjxffthd QA qa.backend.domain.com
   - For other environments remove that prefix
 
 
+### #errorset <subcommand>
+
+### #add
+
+```
+mashery-toolbelt errorset add <serviceId> <errorSetPath>
```

Comment:
I do not reckon there will be different error sets in the future, would it be possible to omit the last parameter?